TeensyDmx.cpp

```diff
@@ -61,7 +34,7 @@ struct DEVICEINFO
   uint16_t startAddress;
   uint16_t subDeviceCount;
   byte sensorCount;
-}; // struct DEVICEINFO
+} __attribute__((__packed__)); // struct DEVICEINFO
```

Comment:
Might be nice to add a static assert to verify the length?

Reply:
What's your preferred way to do this? Do you just use static_assert and only target C++11? In another project I'm involved in, we use this macro:
https://github.com/OpenLightingProject/ola/blob/master/include/ola/base/Macro.h#L91

Reply:
I'm pretty sure that Arduino has supported C++11 for a while, so probably ok to target it only.  I thought I recognised the name, I did a lot with OLA years ago :)

Reply:
Done in a future commit.

Reply:
Done in a future commit.

Reply:
Done in a future commit.

Were you a user or did you do some development on it too? We're always looking for more developers. :) Have you seen our other project Open Lighting Embedded too? https://github.com/OpenLightingProject/ja-rule